### PR TITLE
WIP: Add tensorflow to detect tags.

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

**Comment by patrickelectric** [comment]
@Williangalvani can you review this code ?
The next step will be to integrate this into tag detection algorithm.

File: neural/dataset.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
+import cv2
+import numpy as np
+import os
+
+
+def load(dataset_dir='./dataset', use_y_as_string=False, verbose=True, test_perc=0):
+    '''load all generic data'''
```
Comment (Williangalvani) [inline neural/dataset.py:7]: perhaps as a docstring here? at least explaining what the keyword arguments do?

File: neural/run_tf.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# Build fully connected DNN
+with tf.Session() as session:
+    # Run the initializer
+    session.run(init)
+    with tf.device('/gpu:0'):
+        # Enable logging
+        tf.logging.set_verbosity(tf.logging.INFO)
+        # Create our classifier
+        feature_columns = [tf.contrib.layers.real_valued_column("", dimension=1024)]
+        classifier = tf.contrib.learn.DNNClassifier(feature_columns=feature_columns, hidden_units=[400, 400, 400, 400, 400, 400], n_classes=4, model_dir="model")
```
Comment (Williangalvani) [inline neural/run_tf.py:27]: Have you tried with fewer hidden units? I believe that for a task as simple as this, 1 or no hidden layer at all could be enough.

Comment (Williangalvani) [reply neural/run_tf.py:27]: Depending on how fast it runs, I'd suggest you to do a benchmarking on this and see what happens

**Pull request merged**